sdk/jobs/pipelines/3a_pipeline_with_spark/spark-in-pipeline.ipynb

```diff
@@ -0,0 +1,144 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import required libraries\n",
+    "from azure.identity import DefaultAzureCredential, InteractiveBrowserCredential\n",
+    "from azure.ml import MLClient, Input, Output\n",
```

Comment:
azure.ml -> azure.ai.ml

Reply:
I can't import it from `azure.ai.ml`. I can successfully import it from `azure.ai.ml`.
![image](https://user-images.githubusercontent.com/38863503/180692992-a79bd8b8-33b4-49ce-9a89-7d95afbf72bd.png)

Reply:
Updated based on latest SDK